test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity nodeAffinity
+	InterpodAffinity interpodAffinity
+}
+
+// interpodAffinity priority configuration details.
+type interpodAffinity struct {
+	Enabled     bool
+	Operator    metav1.LabelSelectorOperator
+	affinityKey string
+	Labels      map[string]string
+	TopologyKey string
+}
+
+// nodeAffinity priority configuration details.
+type nodeAffinity struct {
+	Enabled         bool   //If not enabled, node affinity is disabled.
```

Comment:
For that matter interpret nil as "false"?  That way no coupling of high struct to predicate/prio names